tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -735,25 +752,35 @@ engine renders in linear color space.
+| Type                 | Hint                                           | Description                          |
++======================+================================================+======================================+
+| **vec4**             | hint_color                                     | Used as color.                       |
++----------------------+------------------------------------------------+--------------------------------------+
+| **int, float**       | hint_range(min, max[, step])                   | Used as range (with min/max/step).   |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | hint_albedo                                    | Used as albedo color, default white. |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | hint_black_albedo                              | Used as albedo color, default black. |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | hint_normal                                    | Used as normalmap.                   |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | hint_white                                     | As value, default to white.          |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | hint_black                                     | As value, default to black.          |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | hint_anisotropy                                | As flowmap, default to right.        |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | hint_roughness[_r, _g, _b, _a, _normal, _gray] |                                      |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | filter[_nearest, _linear][_mipmap][_aniso]     | Enabled specified texture filtering. |
++----------------------+------------------------------------------------+--------------------------------------+
+| **sampler2D**        | repeat_[enable, disable]                       | Enabled texture repeating.           |
++----------------------+------------------------------------------------+--------------------------------------+
+| <instantiable type>  | instance_index(index)                          |                                      |
++----------------------+------------------------------------------------+--------------------------------------+
```

Comment:
"<instantiable type>" Maybe I need to replace it with the reference to a note, but I don't know how.